modules/proj4/src/lib/proj4-projection.d.ts

```diff
@@ -0,0 +1,24 @@
+import proj4 from 'proj4';
+
+/**
+ * Predefined Aliases
+ * 'WGS84':       'EPSG:4326'
+ * 'EPSG:3785':   'EPSG:3857' 'EPSG:4269'
+ * 'GOOGLE':      'EPSG:3857' 'EPSG:4269'
+ * 'EPSG:900913': 'EPSG:3857' 'EPSG:4269'
+ * 'EPSG:102113': 'EPSG:3857' 'EPSG:4269'
+ */
+export class Proj4Projection {
+  /** Define aliases for one or more projections */
+  static defineProjectionAliases(projections: {
+    [name: string]: string
+  });
+
+  /** Create a projection between two coordinate systems */
+  constructor(projections: {from?: string, to?: string});
+
+  /** Project a coordinate between two coordinate systems */
```

Comment:
Might be a little clearer for the line to say "project from first to second coordinate system", and then for `unproject` to say "project back from second to first coordinate system"?

(I suppose you have existing projection standard functions)